elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-env jquery, browser */
+
+const TABWIDTH = 50;      // defines how many px the answer block is indented by, when the student
+                          // drags and indents a block
+const INDENT_OFFSET = 10;  // For aesthetic, all answer blocks are offseted to the right by
+                          // 5px, so the answer tiles are not directly touching the dropzone margins
+const MAX_INDENT = 4;     // defines the maximum number of times an answer block can be indented
```

Comment:
we could change it to a variable that the user could specify as an HTML attribute. Will having too many attributes the user could configure for `pl-order-blocks` be a problem?

Reply:
This and some of the other comments are really great ideas which can be done as follow-up PRs to this main one. 

It would be good to focus on the necessary items to get this PR merged for now so that people can start using it, and then multiple people can be working on improvements at once after that (e.g. the Proof grading things I want to add).

Reply:
yeah for sure, is there anything else crucial that I should get to adding? I think most of the crucial reviews have been addressed.

Reply:
we have moved these feature suggestions / improvements to issue #3476.